conf/ipa.yaml.template

```diff
@@ -15,3 +15,6 @@ IPA:
   GROUPS:
     - satadmins
     - satusers
+  IDM_PASSWORD:
```

Comment:
Basically you are trying to introduce dubious entry for `IPA:` -> `PASSWORD:`
Stick to `settings.ipa.password`